### PR TITLE
feat: added the count nonzeros function

### PR DESCRIPTION
**Pull request by HildaU**
Added count_nonzeros function #21365

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -133,3 +133,9 @@ def cos(self, name=None):
     @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
     def exp(self, name=None):
         return ivy.exp(self._ivy_array)
+
+
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+def count_non_zeros(self, axis=1, keepdims=True):
+    self = ivy.array(self)
+    return ivy.count_nonzero(self)
```
Comment (zhumakhan) [inline ivy/functional/frontends/paddle/tensor/tensor.py:141]: Tests are not running probably due to `count_nonzero` is not implemented in one of backends: `AttributeError: type object 'Tensor' has no attribute 'count_nonzero'`. Could you please fix it?

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 2]
```diff
@@ -133,3 +133,9 @@ def cos(self, name=None):
     @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
     def exp(self, name=None):
         return ivy.exp(self._ivy_array)
+
+
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+def count_non_zeros(self, axis=1, keepdims=True):
+    self = ivy.array(self)
```
Comment (zhumakhan) [inline ivy/functional/frontends/paddle/tensor/tensor.py:140]: Why this is needed?

**Review by zhumakhan — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/paddle/tensor/tensor.py (hunk 1 above)
Comment (HildaU) [reply ivy/functional/frontends/paddle/tensor/tensor.py]: Hi [zhumakhan], I have made changes to the code, Kindly review it Thanks

**Comment by zhumakhan** [comment]
Hey, everything looks good to me. Could you resolve the conflicts, just want to see tests running?

**Comment by HildaU** [comment]
> Hey, everything looks good to me. Could you resolve the conflicts, just want to see tests running?
Hi zhumakhan, Please kindly review my pr I have made some changes

**Comment by joaozenobio** [comment]
Hi @HildaU ! Thanks for the PR submitted :smile: I am currently looking for an error occurring in the Paddle test suite so we can proceed on correcting your test code. In the meanwhile, please solve the conflicts on your branch and pin me whenever it is done :metal:

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**